src/autoscaler/eventgenerator/config/config_test.go

```diff
@@ -1073,78 +1091,6 @@ defaultBreachDurationSecs: NOT-INTEGER-VALUE
 			})
 		})
 
-		Context("with partial config", func() {
-			BeforeEach(func() {
-				configBytes = []byte(`
-db:
-  policy_db:
-    url: postgres://postgres:password@localhost/autoscaler?sslmode=disable
-  app_metrics_db:
-    url: postgres://postgres:password@localhost/autoscaler?sslmode=disable
-scalingEngine:
-  scaling_engine_url: http://localhost:8082
-metricCollector:
-  metric_collector_url: http://localhost:8083
-defaultStatWindowSecs: 300
-defaultBreachDurationSecs: 600
-`)
-			})
-
-			It("returns default values", func() {
-				Expect(err).NotTo(HaveOccurred())
-				Expect(conf.Aggregator.PolicyPollerInterval).To(Equal(time.Duration(DefaultPolicyPollerInterval)))
-
-				Expect(err).NotTo(HaveOccurred())
-				Expect(conf).To(Equal(&Config{
-					Logging: LoggingConfig{Level: "info"},
-					DB: DBConfig{
-						PolicyDB: db.DatabaseConfig{
-							Url:                   "postgres://postgres:password@localhost/autoscaler?sslmode=disable",
-							MaxOpenConnections:    0,
-							MaxIdleConnections:    0,
-							ConnectionMaxLifetime: 0 * time.Second,
-						},
-						AppMetricDB: db.DatabaseConfig{
-							Url:                   "postgres://postgres:password@localhost/autoscaler?sslmode=disable",
-							MaxOpenConnections:    0,
-							MaxIdleConnections:    0,
-							ConnectionMaxLifetime: 0 * time.Second,
-						},
-					},
-					Aggregator: AggregatorConfig{
-						AggregatorExecuteInterval: DefaultAggregatorExecuteInterval,
-						PolicyPollerInterval:      DefaultPolicyPollerInterval,
-						SaveInterval:              DefaultSaveInterval,
-						MetricPollerCount:         DefaultMetricPollerCount,
-						AppMonitorChannelSize:     DefaultAppMonitorChannelSize,
-						AppMetricChannelSize:      DefaultAppMetricChannelSize,
-					},
-					Evaluator: EvaluatorConfig{EvaluationManagerInterval: DefaultEvaluationExecuteInterval,
-						EvaluatorCount:          DefaultEvaluatorCount,
-						TriggerArrayChannelSize: DefaultTriggerArrayChannelSize},
-					ScalingEngine: ScalingEngineConfig{
-						ScalingEngineUrl: "http://localhost:8082"},
-					MetricCollector: MetricCollectorConfig{
-						MetricCollectorUrl: "http://localhost:8083"},
-					Lock: LockConfig{
-						LockRetryInterval: DefaultRetryInterval,
-						LockTTL:           DefaultLockTTL},
-					DBLock: DBLockConfig{
-						LockTTL:           DefaultDBLockTTL,
-						LockRetryInterval: DefaultDBLockRetryInterval,
-					},
-					EnableDBLock:              false,
-					DefaultBreachDurationSecs: 600,
-					DefaultStatWindowSecs:     300,
-					CircuitBreaker: CircuitBreakerConfig{
-						BackOffInitialInterval:  DefaultBackOffInitialInterval,
-						BackOffMaxInterval:      DefaultBackOffMaxInterval,
-						ConsecutiveFailureCount: DefaultBreakerConsecutiveFailureCount,
-					},
-				}))
-			})
-		})
-
```

Comment:
why remove the test?

Reply:
it is a duplicate of test case:

https://github.com/boyang9527/app-autoscaler/blob/d0c94cab91e8838ba887dde90830a8eb3e0d5ccf/src/autoscaler/eventgenerator/config/config_test.go#L218